src/plugins/external_tool_support/src/bedtools/BedtoolsIntersectTask.cpp

```diff
@@ -199,11 +204,32 @@ Document *BedtoolsIntersectAnnotationsByEntityTask::createAnnotationsDocument(co
     foreach (const U2EntityRef &enRef, entities) {
         U2AnnotationTable t = U2FeatureUtils::getAnnotationTable(enRef, os);
         AnnotationTableObject *table = new AnnotationTableObject(t.visualName, enRef);
+        renameAnnotationsForBed(table->getRootGroup());
         doc->setLoaded(true);
         doc->addObject(table);
     }
 
     return doc;
 }
 
+void BedtoolsIntersectAnnotationsByEntityTask::renameAnnotationsForBed(AnnotationGroup *group) {
+    bool isNumber = false;
+    group->getName().toInt(&isNumber);
+    if (isNumber) {
+        group->setName(group->getName() + RENAME_STRING);
+    }
+    foreach(AnnotationGroup * g, group->getSubgroups()) {
```

Comment:
use for, not foreach

Reply:
Why? What's wrong with foreach?

Reply:
foreach (Qt only) was added when there was no for (C++)

Compare
 foreach(AnnotationGroup * g, group->getSubgroups()) {
vs
 for(AnnotationGroup * g: group->getSubgroups()) {

Tools that not aware about QT may not understand foreach macro (refactoring, highlighting, formatting, etc...)